java/fory-core/src/main/java/org/apache/fory/resolver/ClassResolver.java

```diff
@@ -1802,15 +1794,6 @@ public void ensureSerializersCompiled() {
               }
             }
           });
-      if (GraalvmSupport.isGraalBuildtime()) {
-        classInfoMap.forEach(
-            (cls, classInfo) -> {
-              if (classInfo.serializer != null
-                  && !extRegistry.initialClassInfos.contains(classInfo)) {
-                classInfo.serializer = null;
```

Comment:
Actually we do need the clear, otherwise current Fory instance may hold some unexpected serializer. For example, `ObjectStreamSerializer` hold CompatibleSerializer serializer instead of generated by Fory.

```java
      if (GraalvmSupport.isGraalBuildtime()) {
        // trigger serializer constructor method handle generate.
        Serializers.newSerializer(fory, type, sc);
      }
      if (sc == CompatibleSerializer.class || GraalvmSupport.isGraalBuildtime()) {
        // skip init generated serializer at graalvm build time
        this.slotsSerializer = new CompatibleSerializer(fory, type, fieldResolver);
      } else {
        this.slotsSerializer = (CompatibleSerializerBase) Serializers.newSerializer(fory, type, sc);
      }
```